cmd/kube-apiserver/app/server.go

```diff
@@ -438,33 +456,24 @@ func parseRuntimeConfig(s *options.APIServer) (map[string]genericapiserver.APIGr
 	}
 	_ = disableLegacyAPIs // hush the compiler while we don't have legacy APIs to disable.
 
-	// "api/v1={true|false} allows users to enable/disable v1 API.
```

Comment:
I basically fixed todo from line 457: // TODO: Make this a loop over all group/versions when there are more of them.
and added support for `autoscaling/v1`. The logic should remain the same (except for support for the new api group).